commit 51e45beeb3755bca44d21ef38433013cd41b4017
Author: Andrew John Hughes <gnu_andrew@member.fsf.org>
Date:   Wed Apr 30 21:39:16 2014 +0100

    Use correct inclusion method, allowing builds with FreeType >=2.5.3.
    
    2014-04-20  Andrew John Hughes  <gnu_andrew@member.fsf.org>
    
    	* native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c:
    	Use correct inclusion method, allowing builds with >=2.5.3.
    	* native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c:
    	Likewise.
    
    Signed-off-by: Andrew John Hughes <gnu_andrew@member.fsf.org>

diff --git a/ChangeLog b/ChangeLog
index 4548c1d..e5879eb 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,10 @@
+2014-04-20  Andrew John Hughes  <gnu_andrew@member.fsf.org>
+
+	* native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c:
+	Use correct inclusion method, allowing builds with >=2.5.3.
+	* native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c:
+	Likewise.
+
 2012-11-02  Andrew John Hughes  <gnu_andrew@member.fsf.org>
 
 	PR classpath/55182
--- a/native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c
+++ b/native/jni/gtk-peer/gnu_java_awt_peer_gtk_FreetypeGlyphVector.c
@@ -42,8 +42,9 @@ exception statement from your version. */
 #include <pango/pango.h>
 #include <pango/pangoft2.h>
 #include <pango/pangofc-font.h>
-#include <freetype/ftglyph.h>
-#include <freetype/ftoutln.h>
+#include <ft2build.h>
+#include FT_GLYPH_H
+#include FT_OUTLINE_H
 #include "jcl.h"
 #include "gdkfont.h"
 #include "gnu_java_awt_peer_gtk_FreetypeGlyphVector.h"
diff --git a/native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c b/native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c
index 771b23e..cfd988c 100644
--- a/native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c
+++ b/native/jni/gtk-peer/gnu_java_awt_peer_gtk_GdkFontPeer.c
@@ -39,10 +39,11 @@
 #include <pango/pango.h>
 #include <pango/pangoft2.h>
 #include <pango/pangofc-font.h>
-#include <freetype/ftglyph.h>
-#include <freetype/ftoutln.h>
-#include <freetype/fttypes.h>
-#include <freetype/tttables.h>
+#include <ft2build.h>
+#include FT_GLYPH_H
+#include FT_OUTLINE_H
+#include FT_TYPES_H
+#include FT_TRUETYPE_TABLES_H
 #include "gdkfont.h"
 #include "gtkpeer.h"
 #include "gnu_java_awt_peer_gtk_GdkFontPeer.h"
